File: modul_praktikum_python/contoh_code/pascal/tunjangan_dan_potongan.pas
uses crt; 
var 
  JumlahAnak : integer; 
  GajiKotor,Tunjangan,PersenTunjangan,PersenPotongan : real ; 
  Potongan:Real; 
begin 
  PersenTunjangan:=0.2; 
  PersenPotongan:=0.05; 
  write(‘Gaji Kotor ?’);readln(GajiKotor); 
  write(‘Jumlah Anak ?’);readln(JumlahAnak); 
  if JumlahAnak>2 then  
   begin 
     PersenTunjangan:=0.3; 
     PersenPotongan:=0.07;  
   end; 
  Tunjangan := PersenTunjangan*GajiKotor; 
  Potongan:= PersenPotongan*GajiKotor; 
  writeln(‘Besar Tunjangan = Rp ’,Tunjangan:10:2);   
  writeln(‘Besar Potongan  = Rp ’,Potongan:10:2);   
  readkey;  
end.

File: modul_praktikum_python/contoh_code/python/tunjangan_dan_potongan.py
"""Tunjangan dan potongan gaji berdasarkan jumlah anak."""


def main() -> None:
    persen_tunjangan = 0.2
    persen_potongan = 0.05
    gaji_kotor = float(input("Gaji Kotor ?"))
    jumlah_anak = int(input("Jumlah Anak ?"))
    if jumlah_anak > 2:
        persen_tunjangan = 0.3
        persen_potongan = 0.07
    tunjangan = persen_tunjangan * gaji_kotor
    potongan = persen_potongan * gaji_kotor
    print(f"Besar Tunjangan = Rp {tunjangan:10.2f}")
    print(f"Besar Potongan  = Rp {potongan:10.2f}")
    input("Tekan Enter untuk melanjutkan...")


if __name__ == "__main__":
    main()
